app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+
+  def create
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.create(comment_params)
```

Comment:
@JuanMaGiraldo what if the **create method** for some reason fails?

Reply:
In case of failure, the method will throw an exception and the user would display an error page.

Reply:
@JuanMaGiraldo as we discussed, this needs to be managed through the code to prevent raising errors on the user side